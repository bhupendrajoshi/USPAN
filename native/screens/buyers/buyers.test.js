import 'react-native';
import React from 'react';
import Buyers from './buyers';

import renderer from 'react-test-renderer';

describe('Buyers component', () => {
    test('it should render correctly', () => {
        const tree = renderer.create(
            <Buyers />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
