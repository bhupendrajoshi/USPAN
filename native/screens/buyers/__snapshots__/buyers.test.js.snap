// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buyers component it should render correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    Buyers
  </Text>
</View>
`;
